docs/tutorials/istio.md

```diff
@@ -0,0 +1,28 @@
+# Configure Istio Service Mesh
+
+Integrating Istio's service mesh with Coder's Ingress enables powerful traffic management, security, and observability capabilities. By placing Coder's workspace traffic behind Istio's intelligent proxy layer, you can implement access controls, encrypt service-to-service communication, and gain visibility into your workspace network patterns. This guide walks through the process of configuring Istio alongside Coder's existing ingress controller, ensuring that developer workspaces remain accessible while benefiting from Istio's comprehensive service mesh features.
```

Comment:
```suggestion
Use Istio service mesh for your Coder workspace traffic to implement access controls, encrypt service-to-service communication, and gain visibility into your workspace network patterns.

This configures Istio alongside Coder's existing ingress controller, ensuring that developer workspaces remain accessible while benefiting from Istio's comprehensive service mesh features.
```

Reply:
Coder does not have an existing ingress controller in the Kubernetes sense of the word.

Also, we could probably just drop the first sentence and get straight to the point:

```
This guide walks through the required steps to configure the Istio service mesh for use with Coder.
```